Traceback (most recent call last):
  File "run_ppo_navigation.py", line 417, in <module>
    main(**vars(parser.parse_args()))
  File "run_ppo_navigation.py", line 406, in main
    train_PPO(host,world_port)
  File "run_ppo_navigation.py", line 347, in train_PPO
    a, a_log_prob = prev_policy.choose_action(format_(s).to(device))
AttributeError: 'list' object has no attribute 'to'
